next_sparseconvnet/networks/architectures.py

```diff
@@ -0,0 +1,60 @@
+
+import torch
+import sparseconvnet as scn
+from .building_blocks import ResidualBlock_downsample, ResidualBlock_basic, ResidualBlock_upsample
+
+class UNet(torch.nn.Module):
+        def __init__(self, spatial_size, inplanes, kernel, stride, basic_num, level_depth, dim = 3):
+            torch.nn.Module.__init__(self)
```

Comment:
Is good practice to add docstrings (text inside three quotation marks) where you explain what the class does, input parameters- their types and purpose, and what is the output